lib/phoenix/code_reloader/proxy.ex

```diff
@@ -1,48 +1,120 @@
-# A tiny proxy that stores all output sent to the group leader
-# while forwarding all requests to it.
+# This proxy is spawned at startup and replaces (and forwards to)
+# the standard_error process.
 defmodule Phoenix.CodeReloader.Proxy do
   @moduledoc false
   use GenServer
 
-  def start() do
-    GenServer.start(__MODULE__, "")
+  def start_link() do
+    stderr = Process.whereis(:standard_error)
```

Comment:
I’ve not made any changes wrt. this as I’m unsure the best way to solve it.

Perhaps have the calls to `Proxy.capture` check if it has been started yet and then start it at first call if necessary?